src/plugins/discover/public/application/angular/create_discover_grid_directive.tsx

```diff
@@ -7,25 +7,40 @@
  */
 import React, { useState } from 'react';
 import { I18nProvider } from '@kbn/i18n/react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import { DiscoverGrid, DiscoverGridProps } from '../components/discover_grid/discover_grid';
 import { getServices } from '../../kibana_services';
 import { ElasticSearchHit } from '../doc_views/doc_views_types';
+import { TotalDocuments } from '../apps/main/components/total_documents/total_documents';
+
+export interface DiscoverGridEmbeddableProps extends DiscoverGridProps {
+  totalHitCount: number;
+}
 
 export const DataGridMemoized = React.memo((props: DiscoverGridProps) => (
   <DiscoverGrid {...props} />
 ));
 
-export function DiscoverGridEmbeddable(props: DiscoverGridProps) {
+export function DiscoverGridEmbeddable(props: DiscoverGridEmbeddableProps) {
   const [expandedDoc, setExpandedDoc] = useState<ElasticSearchHit | undefined>(undefined);
 
   return (
     <I18nProvider>
-      <DataGridMemoized
-        {...props}
-        setExpandedDoc={setExpandedDoc}
-        expandedDoc={expandedDoc}
-        services={getServices()}
-      />
+      <EuiFlexGroup style={{ width: '100%' }} direction="column" gutterSize="xs" responsive={false}>
```

Comment:
why is `width: 100%` needed here?

Reply:
Width needs to be provided for discover grid container, othervise the content of the table will be overflowed out of the embeddable view.